docs/pages/performance.md

```
@@ -7,6 +7,113 @@ noindex: true
 sitemap: false
 ---
 
+# {{ page.title }} (Windows, Python 3.13)
+
+These are the execution times for the indicators using two years of historical daily stock quotes (502 periods) with default or typical parameters on Windows.
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)